yaml/ver10/cloudintegration/yaml/cloudintegration.yaml

```diff
@@ -0,0 +1,149 @@
+openapi: 3.1.0
+info:
+  title: Cloud Integration Specification
+  contact:
+    name: ONVIF Technical Specification
+    url: https://github.com/onvif/specs
+    email: help@onvif.org
+  license:
+    name: ONVIF Contributor License Agreement
+    url: https://github.com/onvif/specs/blob/development/LICENSE.md
+  version: "25.06"
+
+components:
+  schemas:
+    Base64DERencodedASN1Value:
+      description: This schema is the equivalent of tas:Base64DERencodedASN1Value in the Security Service specification.
+      type: string
+      format: byte
+      examples: [ Base64DERencodedASN1Value ]
+
+    EndpointReferenceType:
+      description: This schema is the equivalent of wsa:EndpointReferenceType in the WA-Addressing 1.0 specification.
+      type: string
+      format: uri
+
+    AuthorizationServerConfiguration:
+      type: object
+      description: This schema is the equivalent of tas:AuthorizationServerConfigurationData
+      properties:
+        ServerURI:
```

Comment:
We have trust anchors for the Uplink, but not for the authorization server. Are they supposed to be the same? Or should we have another list here? (Although sending a dozen certificates twice might get big fast)

Reply:
As in my previous comment, the device could receive the certificates but it will generate the ids, so the OCS may not be able to define a certification path validation policy.

Reply:
Yeah that's fair.
But right now, we are sending a list of certs associated with the Uplink. This way the device can validate its uplink trust anchor.
But the device also needs to validate the Authorization Server. And there's no trust anchors there. We need to make it clearer, should the trust anchors apply to both of them? Or do we need two lists?